.editorConfig

```diff
@@ -2,3 +2,4 @@ root=true
 [*.{kt,kts}]
 insert_final_newline=true
 max_line_length=140
+disabled_rules=import-ordering
```

Comment:
I understand the rule of `import-ordering` is probably nonsensical and may cause problem (https://github.com/pinterest/ktlint/issues/ 527), but looks like it doesn't work as expected according to the result the `Test Workflow`
There are still errors in the log: `Imports must be ordered in lexicographic order without any empty lines in-between`.